Sources/Clappr/Classes/Base/Layers/LayerComposer.swift

```diff
@@ -35,4 +35,16 @@ public class LayerComposer {
     func attachOverlay(_ view: UIView) {
         overlayLayer.attachOverlay(view)
     }
+    
+    func showViews() {
+        mediaControlLayer.isHidden = false
+        coreLayer.isHidden = false
+        overlayLayer.isHidden = false
+    }
+
+    func hideViews() {
+        mediaControlLayer.isHidden = true
+        coreLayer.isHidden = true
+        overlayLayer.isHidden = true
+    }
```

Comment:
I suggest using the more descriptive names: `showUI` and `hideUI`. There are no parameters to these methods, so its hard to now which views are we talking about. As we are only hiding views with user interaction, I guess the `UI` is more appropriate.